frontend/packages/erd-core/src/components/ERDRenderer/ErrorDisplay.module.css

```diff
@@ -1,8 +1,161 @@
-/* TODO: styling */
 .wrapper {
-  position: relative;
-  width: 100%;
+  font-family: var(--main-font, Inter);
+  display: flex;
+  padding: 0px var(--spacing-6, 24px);
+  flex-direction: column;
+
   height: 100%;
+
+  justify-content: center;
+  align-items: center;
+  gap: var(--spacing-10, 40px);
+
   background-color: var(--global-background);
-  color: blanchedalmond;
+}
+
+.main {
+  display: flex;
+  padding: var(--spacing-8, 32px);
+  align-items: flex-start;
+  gap: var(--spacing-6, 24px);
+
+  border-left: 1px solid var(--callout-warning-border, rgba(255, 215, 72, 0.2));
+  background: var(--callout-warning-background, rgba(255, 215, 72, 0.1));
+}
+
+.iconWrapper {
+  display: flex;
+  width: 24px;
+  padding-top: var(--spacing-1, 4px);
+  align-items: center;
+}
+
+.inner {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  gap: var(--spacing-8, 32px);
+}
+
+.message1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--spacing-2, 8px);
+}
+
+.message1Title {
+  color: var(--callout-warning-text, #c3b476);
+  font-family: var(--message-font, Montserrat);
+  font-size: var(--font-size-8, 20px);
+  font-weight: 700;
+  line-height: 160%;
+}
+
+.message1Sentence {
+  color: var(--callout-warning-text, #c3b476);
+  font-size: var(--font-size-5, 14px);
+  font-weight: 400;
+  line-height: 160%;
+}
+
+.message2 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--spacing-2, 8px);
+}
+
+.message2Title {
+  color: var(--callout-warning-text, #c3b476);
+  font-size: var(--font-size-7, 18px);
+  font-weight: 700;
+  line-height: 160%;
+}
+
+.message2Sentence {
+  color: var(--callout-warning-text, #c3b476);
+
+  font-size: var(--font-size-5, 14px);
+  font-weight: 400;
+  line-height: 160%;
+}
+
+.callout {
+  color: var(--callout-warning-link, #e7ddb3);
+  font-size: var(--font-size-5, 14px);
+  font-weight: 400;
+  line-height: normal;
+
+  text-decoration: underline;
+  text-decoration-color: var(
+    --callout-warning-underline,
+    rgba(231, 221, 179, 0.4)
+  );
+}
+
+.message3 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: var(--spacing-2, 8px);
+  align-self: stretch;
+}
+
+.message3Title {
+  color: var(--callout-warning-text, #c3b476);
+  font-size: var(--font-size-5, 14px);
+  font-weight: 700;
+  line-height: 160%;
+}
+
+.message3Sentence {
+  color: var(--callout-warning-text, #c3b476);
+
+  font-size: var(--font-size-5, 14px);
+  font-weight: 400;
+  line-height: 160%;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
Thank you for the responsive support!
However, the design for the mobile version is not yet complete and will be reimplemented later. But looks like it will stay that way for now.

Reply:
You're absolutely right! There’s a chance the web designer's intentions and design choices might not be fully captured, so I’ll add a comment to the CSS to reflect this.